pkg/controller/fileintegrity/fileintegrity_controller.go

```diff
@@ -23,6 +23,143 @@ import (
 
 var log = logf.Log.WithName("controller_fileintegrity")
 
+const (
+	fileIntegrityNamespace     = "openshift-file-integrity"
+	defaultConfDataKey         = "aide.conf"
+	defaultConfigMapName       = "aide-conf"
+	workerDaemonSetName        = "aide-worker"
+	masterDaemonSetName        = "aide-master"
+	aideScriptConfigMapName    = "aide-script"
+	operatorServiceAccountName = "file-integrity-operator"
+	aideScriptPath             = "/scripts/aide.sh"
+)
+
+var aideScript = `
+    #!/bin/sh
+    if test ! -f /hostroot/etc/kubernetes/aide.db.gz; then
+      echo "initializing AIDE db"
+      aide -c /tmp/aide.conf -i
+    fi
+    while true; do
+      echo "running AIDE check.."
+      aide -c /tmp/aide.conf
+      echo "AIDE check returned $?.. sleeping"
+      sleep 5m
+    done
+    exit 1
+`
+
+var defaultAideConfig = `
+    @@define DBDIR /hostroot/etc/kubernetes
+    @@define LOGDIR /hostroot/etc/kubernetes
```

Comment:
Logdir should probably not be in `/etc/` either, I'm not sure if SELinux would like us writing to `/etc/`, also logrotate typically doesn't touch anything under `etc/`.

btw regarding logrotate, should we rotate the logs? I noticed that `report_url` is configured so that the logs are both written to the disk and to stdout

Reply:
FIled https://github.com/mrogers950/file-integrity-operator/issues/4 and https://github.com/mrogers950/file-integrity-operator/issues/5 for these queries.